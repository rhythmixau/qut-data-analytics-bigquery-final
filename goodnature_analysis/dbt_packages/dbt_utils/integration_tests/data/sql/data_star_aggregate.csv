version https://git-lfs.github.com/spec/v1
oid sha256:1b49cfd6ffe62abb23ac1488cb2e376ba005cb023868c499b571ad765b3aa6d9
size 63
