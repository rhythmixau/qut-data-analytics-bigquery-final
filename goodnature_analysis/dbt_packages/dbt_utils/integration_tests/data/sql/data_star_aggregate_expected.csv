version https://git-lfs.github.com/spec/v1
oid sha256:52b690707a14b3ea472506f1f7d296b5cb60c6c16d149c1b9285e119ea5fb1f5
size 61
